tests/integration/test_fim/test_files/test_recursion_level/test_recursion_level.py

```diff
@@ -21,15 +21,17 @@
 dir_no_recursion = os.path.join(PREFIX, 'test_no_recursion')
 dir_recursion_1 = os.path.join(PREFIX, 'test_recursion_1')
 dir_recursion_5 = os.path.join(PREFIX, 'test_recursion_5')
-dir_recursion_320 = os.path.join(PREFIX, 'test_recursion_320')
+dir_recursion_320 = os.path.join(PREFIX, 'test_recursion_32') if sys.platform == "win32" else os.path.join(PREFIX, 'test_recursion_320')
```

Comment:
Change the name of the variable to a generic name  because this can be confusing.